api/src/main/java/org/openmrs/Relationship.java

```diff
@@ -11,25 +11,48 @@
 
 import java.util.Date;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+
+
 /**
  * Relationship
  */
+   @Entity
+   @Table(name = "relationship")
 public class Relationship extends BaseChangeableOpenmrsData {
 	
 	public static final long serialVersionUID = 323423L;
 	
 	// Fields
-	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name="relationship_id",nullable=false)
```

Comment:
This comma should be changed
`
@Column(name="relationship_id", nullable=false)`

Reply:
idin't get you Mr Tendo,atleast a hint about how i should change the comma.can i just get rid of it + the nullable........?

Reply:
@joelss-tech , 
the long term solution is to setup the [OpenMRS formatter](https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse#DeveloperHowToSetupAndUseEclipse-XMLCodeFormat) for your IDE.
you can set it up in Ecclipse, IntelliJ  or VScode